retrace-server.spec.in

```diff
@@ -106,6 +108,7 @@ mkdir -p %{buildroot}%{_libexecdir}/%{name}/hooks/post_clean_task
 
 %if 0%{?fedora} >= 29
 %py_byte_compile %{__python3} %{buildroot}%{_datadir}/%{name}/plugins
+%py_byte_compile %{__python3} %{buildroot}%{_localstatedir}/lib/retrace/plugins
```

Comment:
This isn't neede. There is no /var/lib/retrace/plugins nor there are any python files.